ui/src/app/pages/homepage/homepage.tsx

```diff
@@ -390,24 +402,47 @@ export const Homepage = withUserProfile()(class extends React.Component<
                               <RecentWorkspaces />
                             </FlexColumn>
                             <FlexColumn>
-                              <SmallHeader>Recently Accessed Items</SmallHeader>
-                              <RecentResources/>
+                              {this.state.userHasWorkspaces ? (<React.Fragment>
+                                <SmallHeader>Recently Accessed Items</SmallHeader>
+                                <RecentResources/>
+                              </React.Fragment>) : <div style={{backgroundColor: colors.lightBackground,
+                                color: colors.primary,
+                                borderRadius: 10,
+                                padding: '10px 30px',
+                                margin: '30px 0px'}}>
+                                <h2 style={{fontWeight: 600, marginTop: 0}}>Here are some tips to get you started:</h2>
+                                <CustomBulletList>
+                                  <CustomBulletListItem bullet='→'>
+                                    Create a <a href='https://support.google.com/chrome/answer/2364824'>Chrome Profile </a>
+                                    with your All of Us Researcher workbench Google account. This will keep your Workbench
+                                    browser sessions isolated from your other Google accounts.
+                                  </CustomBulletListItem>
+                                  <CustomBulletListItem bullet='→'>
+                                    Check out <a onClick={() => navigate(['library'])}> Featured Workspaces </a>
```

Comment:
not your code or within the scope of this PR -- this link doesn't play nicely with command+click to open in a new tab -- something we need to refactor in the app.

Reply:
yeah... this is what prompted my discussion at the eng-sync yesterday.

Reply:
Note: here is the react conversion context I was mentioning: https://github.com/all-of-us/workbench/pull/1893#r265816213